install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -13,6 +13,7 @@ data:
       "clusterAPIControllerBareMetal": "quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0",
       "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0",
       "clusterAPIControllerGCP": "quay.io/openshift/origin-gcp-machine-controllers:v4.0.0",
+      "clusterAPIControllerOvirt": "quay.io/rgolangh/cluster-api-provider-ovirt:v4.3",
```

Comment:
can we please ensure the `cluster-api-provider-ovir` goes through the ART build process and add the appropriate imageStream to the images references here

Reply:
@enxebre Here is the release PR for the image https://github.com/openshift/release/pull/6151

Reply:
According to the ART job that tries to create the images -
` error: unable to create a release: operator "machine-api-operator" contained an invalid image-references file: no input image tag named "ovirt-machine-controllers" `
I think that this means that we must merge this first and then the job above-mentioned? @enxebre